modules/cas/lib/Auth/Source/CAS.php

```diff
@@ -78,6 +78,29 @@ public function __construct($info, $config) {
 		}else{
 			throw new Exception("cas login URL not specified");
 		}
+
+
+		// Check for the required ldap configuration 
+		if(!isset($this->_ldapConfig['servers'])) {
+			throw new Exception("ldap server not specified");
+		}
+
+		// Check for ldap options in cas configuration if not configured set to default
```

Comment:
I re-work this and try to find a better way of setting the defaults for the the LDAP constructor from the CAS module, or change the LDAP constructor.